### ISSUE TITLE
Combat report should not say "killing" against multi-hitpoint monsters

### ISSUE THREAD

**Issue by sparr** [artyomtrityak/Atlantis#36]
`Unit (462) shoots a Fireball, killing 1.`

This against a Roc which has 15hp. It was not killed, just reduced to 14hp.

**Comment by artyomtrityak** [comment]
yes, same for multi-health units, it should be changed to dealing 1 damage. Thanks for reporting!
